oidc/verify.go

```diff
@@ -13,14 +13,19 @@ import (
 	"time"
 
 	"golang.org/x/oauth2"
-	jose "gopkg.in/square/go-jose.v2"
```

Comment:
nit: please leave this line alone. I think adding this is the default goimports functionality.

Reply:
Sure thing. Gonna fight with GoLand real quick.  :-)